README.md

```diff
@@ -53,6 +54,8 @@ To get a local copy up and running, follow these steps.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+You can deploy this project using: https://cnualartc.github.io/My_Portfolio/
```

Comment:
[OPTIONAL] I think this block of comments can be more descriptive😉 that link is not a way to deploy the project but rather a link to your life project😉 you can improve the statement by saying `You can view this project live using: https://cnualartc.github.io/My_Portfolio/`  instead🤗

Reply:
Great! thank you for the feedback @Ibizugbe 
I will make the change suggested, push and merge.👍🏼